changelogs/fragments/75062-vault-map-files.yml

```diff
@@ -0,0 +1,3 @@
+major_changes:
+  - Added new method to decrypt multiple named vault files by using a so-called vault map file 
+  - Added new parameter "--vault-map-file" to the vault CLI arguments
```

Comment:
just use the existing and detect the format.

Reply:
I implemented the changes you requested. You wrote that this would require docs. Should I write those and if yes, could you please tell me how to do that?

![Screenshot from 2021-06-26 17-21-34](https://user-images.githubusercontent.com/16357184/123517743-fea1e180-d6a2-11eb-8729-e827d5aa2985.png)

Reply:
Please note - the passing of the vault map file for the separate vault IDs is not necessary but can be used to be more explicit within the CLI params